addons/monit/monit_configurations/monit_general.tt

```diff
@@ -22,11 +22,13 @@ A copy of this alert have been sent to the PacketFence support team.
 # - Do not alert on 'nonexist' because we alert on 'restart'
 # - Do not alert on 'pid' because we alert on 'restart
 # - Do not alert on 'content' because there is a separate alert configuration with reminders
+# - Do not alert on 'connection' because there is a separate alert configuration with reminders
 [% FOREACH email IN EMAILS %]
-    set alert [% email %] but not on { resource nonexist pid content }
-    set alert [% email %] { resource } with reminder on 450 cycle   # reminder every 15 minutes (450 cycles) if 1 cycle is 2 seconds
+    set alert [% email %] but not on { resource nonexist pid content connection }
+    set alert [% email %] { resource } with reminder on 450 cycle       # reminder every 15 minutes (450 cycles) if 1 cycle is 2 seconds
```

Comment:
Just to make sure, is it normal you added 'connection' there ?
Is it a monit keyword ?
And that looks like it will flush the alerts on the connection alerts. If so, I don't see the point of that test at all